src/pages/ShareCodePage.tsx

```diff
@@ -46,13 +54,24 @@ function ShareCodePage({report}: ShareCodePageProps) {
                 shouldShowBackButton
             />
             <ScrollView style={[themeStyles.flex1, themeStyles.pt3]}>
+                <View style={[themeStyles.workspaceSectionMobile, themeStyles.ph5]}>
                 {/* 
                     Right now QR code download button is not shown anymore
                     This is a temporary measure because right now it's broken because of the Fabric update.
                     We need to wait for react-native v0.74 to be released so react-native-view-shot gets fixed.
                     
                     Please see https://github.com/Expensify/App/issues/40110 to see if it can be re-enabled.
-                */}
+                */}                    
+                <QRShare
+                    ref={qrCodeRef}
+                    url={url}
+                    title={title}
+                    subtitle={title}
```

Comment:
It looks like in https://github.com/Expensify/App/pull/40109/files there was some pretty unique `subtitle` logic, do we want to skip adding that for now?

Reply:
Nice catch! I've update code.